extension_templates/forecasting.py

```diff
@@ -261,6 +264,31 @@ def _predict_quantiles(self, fh, X=None, alpha=0.5):
         """
         # implement here
 
+    def _predict_var(self, fh, X=None, cov=False):
+        """
+        Compute/return prediction variance for a forecast.
+
+        Must be run *after* the forecaster has been fitted.
+
+        Parameters
+        ----------
+        fh : int, list, np.array or ForecastingHorizon
+            Forecasting horizon
+        X : pd.DataFrame, optional (default=None)
+            Exogenous time series
+        cov : bool, optional (default=False)
+            Wether to return marginal variances or covariance matrices.
+
+        Returns
+        -------
+        pred_var : np.ndarray
```

Comment:
come to think of it, it should probably be a `DataFrame`, where row indices (and in the case of `cov=True`, column indices) are those of `fh`

Reply:
So no `pd.Series` at all ?

Reply:
Done, it's `Dataframe` now